src/cozy/wrappers/import-page/import-page.component.ts

```diff
@@ -11,6 +28,7 @@ interface ImportPageProps {
     bitwardenData: {
         extension_installed: boolean;
     };
+    vaultData: any;

```

Comment:
Can't we be more specific than just `any`?

Reply:
The type has been defined in `cozy-keys-lib` : https://github.com/cozy/cozy-keys-lib/pull/152/files
But it does not generate `.d.ts` files so I can't target this type from `cozy-pass-web`.

Should I recreate this type on `cozy-pass-web` side?